### ISSUE TITLE
storing dipoles as binary file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [adda-team/adda#90]
```
What new or enhanced feature are you proposing?

for very large target it is not possible to store
the dipoles and/or the internal field as text.
A binary format (better if machine-independent, netcdf) is required. 

What goal would this enhancement help you achieve?

Near field calculations for large target
```

Original issue reported on code.google.com by `fabio.de...@gmail.com` on 23 Nov 2009 at 2:01
- Blocking: #10
- Blocked on: #183

**Comment by myurkin** [comment]
This is a duplicate of #31
